docs/apis-tools/tasklist-api-rest/controllers/tasklist-api-rest-task-controller.md

```diff
@@ -324,7 +440,7 @@ curl -X 'POST' \
 
 #### Responses
 
-| HTTP status | Description                                                        | Response schema                                                                                       |
-| ----------- | ------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------- |
-| 200         | On success                                                         | JSON array of objects with [`VariableResponse`](../schemas/responses/variable-response.mdx) structure |
-| 404         | An error is returned when the task with the `taskId` is not found. | JSON object with [`Error`](../schemas/responses/error-response.mdx) structure                         |
+| HTTP status | Description                                                        | Response schema                                                                                                    |
+| ----------- | ------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------ |
+| 200         | On success                                                         | JSON array of objects with [`VariableSearchResponse`](../schemas/responses/variable-search-response.mdx) structure |
```

Comment:
Would this be considered a breaking change since the name changed?

Reply:
No, this wouldn't be considered a breaking change. The names are more like pseudonyms for the JSON schemas, and clients do not utilize these names.